docs: all-flash mode (#129)
iomesh/docs: docs/deploy/setup-iomesh.md

```

#### Mount Type

> In `mixed-flash` deployment node, IOMesh provides 2 mount types:

Reviewer: mixed -> hybrid

Reviewer: node -> mode

> In `hybrid-flash` deployment mode, IOMesh provides 2 mount types:

Reviewer 2: 一般1-9数字用单词表示，看是否需要将2改成two？如果统一采用阿拉伯数字，那文档中其他1-9的数字也需要改为阿拉伯数字。

> IOMesh provides 2 mount types now:

Reviewer 2: 同上

> - `cacheWithJournal`: used for performance tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommended.

Reviewer 2: 这里的two partitions数字用的是单词，看是否需要全文保持一致。

> - `cacheWithJournal`: used for performance tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommended.

Reviewer 2: another 改为 the other，one and the other 表示二选一，another 表示三者或三者以上。

> - `cacheWithJournal`: used for performance tier of storage pool. It **MUST** be a partitionable block device. IOMesh will partition the device into 2 partitions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommended.

Reviewer: `IOMesh will partition the device into 2 partitions` -> `Two partitions will be created`

- `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommended.

> In `all-flash` deployment mode, IOMesh only provides 1 mount type:

Reviewer 2: 这里数字采用的是阿拉伯数字，看是否需要全文保持一致。

> In `all-flash` deployment node, IOMesh only provides 1 mount type:

Reviewer: node -> mode

> - `dataStoreWithJournal`: used for capacity tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and the other for `cache`. Either `SATA` or `NVMe` SSD is recommended.

Reviewer 3: `recommended` 
It is not recommended, but MUST be either SATA or NVMe SSD?

> - `dataStoreWithJournal`: used for capacity tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and the other for `cache`. Either `SATA` or `NVMe` SSD is recommended.

Reviewer 4: - `dataStoreWithJournal`: used for capacity tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and the other for `dataStore`. Either `SATA` or `NVMe` SSD is recommended.

```

#### Mount Type
In `mixed-flash` deployment node, IOMesh provides 2 mount types:

> - `cacheWithJournal`: used for performance tier of storage pool. It **MUST** ba a partitionable block device. IOMesh will partition the device into 2 paritions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommanded.

Reviewer 5: **MUST** ba --> **MUST** be
paritions --> partitions
one for `journal` and another for `cache`. --> one for `journal` and the other for `cache`.

> - `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommanded.

Reviewer 5: recommanded --> recommended

In `all-flash` deployment node, IOMesh only provides 1 mount type:

- `dataStoreWithJournal`: used for capacity tier of storage pool. It **MUST** ba a partitionable block device. IOMesh will partition the device into 2 paritions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommanded.

> 

Reviewer 5: **MUST** ba --> **MUST** be
2 paritions --> 2 partitions
one for `journal` and another for `cache` --> one for `journal` and the other for `dataStore`

All block devices selected by device selector will be mounted to IOMesh with the corresponding mount type.

> #### Example of deviceMap configuration in mixed-flash deployment node

Reviewer 5: node --> mode

> #### Example of deviceMap configuration in mixed-flash deployment mode

Reviewer: mixed -> hybrid

> #### Example of deviceMap configuration in hybrid-flash deployment mode

Reviewer 2: 这里标题是否需要和前文标题的大小写保持一致？改为：Example of DeviceMap Configuration in Hybrid-Flash Deployment Mode

        - blockdevice-097b6628acdcd83a2fc6a5fc9c301e01
    # ...
```

> #### Example of deviceMap configuration in all-flash deployment mode

Reviewer 2: 同上

```yaml
spec:
  # ...
  chunk:
    # ...
    deviceMap:
      dataStoreWithJournal:
        selector:
          matchLabels:
            iomesh.com/bd-deviceType: disk
          matchExpressions:
          - key: iomesh.com/bd-driverType
            operator: In
            values:
            - SSD
        exclude:
        - blockdevice-097b6628acdcd83a2fc6a5fc9c301e01
    # ...
```

> > **_NOTE_: The blockdevice used by IOMesh cannot have an existing filesystem, please pay attention to the `FSTYPE` field output by the command `kubectl --namespace iomesh-system -owide get blockdevice`.**

Reviewer 2: filesystem后逗号改为句号，please首字母大写Please。

Author: done

> > **_NOTE_: The blockdevice used by IOMesh cannot have an existing filesystem. Please pay attention to the `FSTYPE` field output by the command `kubectl --namespace iomesh-system -owide get blockdevice`.**

Reviewer: ```suggestion
> **_NOTE_: The blockdevice used by IOMesh should not contain an existing filesystem. Please pay attention to the `FSTYPE` field output by the command `kubectl --namespace iomesh-system -owide get blockdevice`.**
```

@yuqiu123 Please double check.

Reviewer 2: That‘s fine.

    # ...
```

> #### Example of deviceMap configuration in all-flash deployment node

Reviewer 5: node --> mode
